examples/stable-diffusion/training/README.md

```diff
@@ -126,86 +126,87 @@ Install the requirements:
   --gaudi_config_name Habana/stable-diffusion \
   --throughput_warmup_steps 3 \
-  --use_hpu_graphs \
-  --bf16
+  --dataloader_num_workers 8 \
+  --bf16 \
+  --use_hpu_graphs_for_training \
+  --use_hpu_graphs_for_inference \
+  --validation_prompt="a robotic cat with wings" \
+  --validation_epochs 48 \
+  --checkpointing_steps 2500 \
+  --logging_step 10
 ```
 
-### Example for LoRA SDXL
-
-Low-Rank Adaption (LoRA) allows adapting a pretrained model by adding pairs of rank-decomposition matrices to
-existing weights and only training those newly added weights.
 
-We can launch the LoRA based fine-tuning of SDXL model using:
+### Multi-card Training
+```bash
+PT_HPU_RECIPE_CACHE_CONFIG=/tmp/stdxl_recipe_cache,True,1024  \
+python ../gaudi_spawn.py --world_size 8 --use_mpi train_text_to_image_sdxl.py \
+  --pretrained_model_name_or_path stabilityai/stable-diffusion-xl-base-1.0 \
+  --pretrained_vae_model_name_or_path stabilityai/sdxl-vae \
+  --dataset_name lambdalabs/pokemon-blip-captions \
+  --resolution 512 \
+  --crop_resolution 512 \
+  --center_crop \
+  --random_flip \
+  --proportion_empty_prompts=0.2 \
+  --train_batch_size 16 \
+  --max_train_steps 336 \
+  --learning_rate 1e-05 \
+  --max_grad_norm 1 \
+  --lr_scheduler constant \
+  --lr_warmup_steps 0 \
+  --output_dir sdxl-pokemon-model \
+  --gaudi_config_name Habana/stable-diffusion \
+  --throughput_warmup_steps 3 \
+  --dataloader_num_workers 8 \
+  --bf16 \
+  --use_hpu_graphs_for_training \
+  --use_hpu_graphs_for_inference \
+  --validation_prompt="a robotic cat with wings" \
+  --validation_epochs 48
+```
 
+### Single-card Training on Gaudi1
 ```bash
-python train_text_to_image_sdxl.py \
-  --pretrained_model_name_or_path="stabilityai/stable-diffusion-xl-base-1.0" \
-  --pretrained_vae_model_name_or_path="madebyollin/sdxl-vae-fp16-fix" \
-  --dataset_name="lambdalabs/pokemon-blip-captions" \
-  --caption_column="text" \
-  --resolution=1024 --random_flip \
-  --train_batch_size=1 \
-  --num_train_epochs=2 --checkpointing_steps=500 \
-  --learning_rate=1e-04 --lr_scheduler="constant" --lr_warmup_steps=0 \
-  --seed=42 \
-  --output_dir="sd-pokemon-model-lora-sdxl" \
-  --finetuning_method="lora" \
-  --gaudi_config_name="Habana/stable-diffusion" \
-  --throughput_warmup_steps=3 \
-  --use_hpu_graphs \
+PT_HPU_MAX_COMPOUND_OP_SIZE=5 python train_text_to_image_sdxl.py \
+  --pretrained_model_name_or_path stabilityai/stable-diffusion-xl-base-1.0 \
+  --pretrained_vae_model_name_or_path stabilityai/sdxl-vae \
+  --dataset_name lambdalabs/pokemon-blip-captions \
+  --resolution 512 \
+  --center_crop \
+  --random_flip \
+  --proportion_empty_prompts=0.2 \
+  --train_batch_size 1 \
+  --gradient_accumulation_steps 4 \
+  --max_train_steps 3000 \
+  --learning_rate 1e-05 \
+  --max_grad_norm 1 \
+  --lr_scheduler constant \
+  --lr_warmup_steps 0 \
+  --output_dir sdxl-pokemon-model \
+  --gaudi_config_name Habana/stable-diffusion \
+  --throughput_warmup_steps 3 \
   --bf16
```

Comment:
can we add a known issue about long graph compilation here for first 2 steps

Reply:
done